pymilvus_orm/index.py

```diff
@@ -104,6 +104,23 @@ def field_name(self) -> str:
         """
         return self._field_name
 
+    def __eq__(self, other) -> bool:
+        """
+        The order of the fields of index must be consistent.
+        """
+        return self.to_dict() == other.to_dict()
+
+    def to_dict(self):
+        """
+        Put collection name, field name and index params into dict.
+        """
+        _dict = {
+            "collection": self._collection._name,
+            "field": self._field_name,
+            "index_param": self._index_params
```

Comment:
Please use `self.params` which is a deep copy of self._index_params, it seems safe